src/Cli/TerminusCommands.php

```diff
@@ -0,0 +1,158 @@
+        'update-releases' => true,
+        'terminus-release' => 'latest',
+        'github-repo' => 'pantheon-systems/documentation',
+        'terminus-repo' => 'pantheon-systems/terminus',
+        'base-branch' => 'main',
+        'branch-name-prefix' => 'docs-update-terminus-',
+        'commit-message' => 'Update terminus information.',
+        'pr-body' => '',
+        'pr-title' => '[UpdateTool - Terminus Information] Update commands and releases.',
+        'dry-run' => false,
+    ])
+    {
+        $api = $this->api($options['as']);
+        $updateCommands = $options['update-commands'];
+        $updateReleases = $options['update-releases'];
+        if (!$updateCommands && !$updateReleases) {
+            throw new \Exception("Either commands or releases must be updated.");
+        }
+
+        $terminusRepo = $options['terminus-repo'];
+        $terminusRelease = $options['terminus-release'];
+
+        $terminusDir = sys_get_temp_dir() . '/update-tool/terminus/';
+        if (!is_dir($terminusDir)) {
+            mkdir($terminusDir, 0755, true);
+        }
+
+        if ($terminusRelease === 'latest') {
+            $terminusRelease = $this->getLatestRelease($api, $terminusRepo);
+        }
+        $downloadUrl = 'https://github.com/pantheon-systems/terminus/releases/download/' . $terminusRelease . '/terminus.phar';
+
+        file_put_contents($terminusDir . 'terminus.phar', file_get_contents($downloadUrl));
+        if (!file_exists($terminusDir . 'terminus.phar')) {
+            throw new \Exception(sprintf('Failed to download terminus.phar from release %s.', $terminusRelease));
+        }
+        chmod($terminusDir . 'terminus.phar', 0755);
+
+        $githubRepo = $options['github-repo'];
+        $baseBranch = $options['base-branch'];
+
+        $branchNamePrefix = $options['branch-name-prefix'];
+        $branchName = $branchNamePrefix . date('Ymd');
+
+        $url = 'git@github.com:' . $githubRepo . '.git';
+        $remote = new Remote($url);
+        $dir = sys_get_temp_dir() . '/update-tool/' . $remote->project();
+        $workingCopy = WorkingCopy::cloneBranch($url, $dir, $baseBranch, $api);
+        $workingCopy->setLogger($this->logger);
+        $workingCopy->createBranch($branchName, $baseBranch, true);
+
+        if ($updateCommands) {
+            $this->logger->info('Updating Terminus commands...');
+            exec(sprintf("cd %s && %s/terminus.phar list --format=json > source/data/commands.json", $dir, $terminusDir), $output, $return);
+            if ($return != 0) {
+                throw new \Exception(sprintf('Failed to list terminus commands. Output: %s', implode("\n", $output)));
+            }
+
+            $commands = json_decode(file_get_contents($dir . '/source/data/commands.json'), true);
+            $commandsJson = json_encode($commands, JSON_PRETTY_PRINT);
+
+            // Adjust output.
+            $commandsJson = str_replace(
+                [
+                'site_env',
```

Comment:
I think the `sed` command I was using was converting from `<site_env>` to `<site>.<env>` for consistency.

Reply:
Fixed